torch/_inductor/codegen/cuda/gemm_template.py

```diff
@@ -606,7 +605,9 @@ def gen_ops(self) -> "List[cutlass_gemm_op.GemmOperation]":  # type: ignore[name
             num_3x_ops,
             num_2x_ops,
         )
-        return list(res.values())[: inductor_cuda_config.cutlass_max_profiling_configs]
+        all_results = list(res.values())
+        random.shuffle(all_results)
```

Comment:
Curious what do we want to achieve by this? This introduces non-determinism which is likely not good?